applications/openshift/api-server/api_server_tls_cipher_suites/kubernetes/shared.yml

```diff
@@ -0,0 +1,12 @@
+---
+# platform = multi_platform_ocp
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  tlsSecurityProfile:
+    type: Custom
+    custom:
+      ciphers: [ {{ .var_apiserver_tls_cipher_suites }} ]
+      minTLSVersion: VersionTLS12
```

Comment:
I wonder if we should parameterize this, too? I'm thinking about cases where someone might want to force the cluster to use PQC-safe algorithms only available through TLS 1.3.